Modules/HR/Config/config.php

```diff
@@ -127,4 +127,11 @@
         'max-reserved-allowed' => 3,
     ],
     'offer-letters-dir' => 'offer-letters',
+    'reasons-for-rejections' => [
+        'No response',
+        'Skills mismatch',
+        'Culture mismatch',
+        'Salary expectation mismatch',
+        'Not enough knowledge/inclination for ColoredCow'
```

Comment:
@kickb71 I think we should keep key=>value pair in the config, and store the slug in the database, since the labels may change in the future. Let me know your thoughts on it.

Reply:
If we change the key value means it will be hard. In for loop in the blade file and javascript file I have to change in both places. If we use a new value means we should just add here the new value but if we use the key-value means we have to change all the files for future use. 
@P4NK4J  I think it should be the same. If it should be changed means I happy to change it

Reply:
@kickb71 , I am saying the `key` will be saved in the database, which would never get changed. If there is any change to the reason let's say `No response` is changed to `No response given`, that would be easy to make only in the config, without changing it from the database, hence recommending to use the following
```suggestion
        'no-response' => 'No response',
        ...
```

Reply:
The same thing happens without the key. I don't know the difference